main.py

```diff
@@ -1,8 +1,13 @@
 import flask
+from flask_cors import CORS
 from src.views import views
 
 app = flask.Flask(__name__)
 app.register_blueprint(views)
 
+
+CORS(app, resources={r"/auth/*": {"origins": "http://localhost:5173"}})
```

Comment:
Enable CORS for all the endpoints and receive the origins as a comma-separated string from the environment variables.